hs_core/page_processors.py

```diff
@@ -71,6 +71,7 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
                         if tool_url.endswith('/'):
                             tool_url = tool_url[:-1]
                         tl = {'title': str(res_type.content_object.title),
+                              'icon': res_type.content_object.tool_icon.first().icon,
```

Comment:
probably should be tool_res_obj.metadata.title and tool_res_obj.metadata.tool_icon... rather than res_type.metadata